### ISSUE TITLE
[18GA] only ACL should be able to trace a route through Jacksonville

### ISSUE THREAD

**Issue by philcampeau** [tobymao/18xx#11206]
[Gist link](https://gist.github.com/philcampeau/8d85cfc90cb0ff0356d4a187071f797b) showing the issue.

**How to Reproduce the Bug?**

As per the rules: 

![image](https://github.com/user-attachments/assets/5522271c-d219-464e-bb62-f918331962bf)

However, if you build a route to Jacksonville before ACL is floated, you can build past it and run trains through it:

![image](https://github.com/user-attachments/assets/39ca743f-eaa8-4282-b7ce-6683bfb13cfd)

**Comment by philcampeau** [comment]
Probably the simplest solution here is to put a blocking token in this city until ACL floats.

**Comment by perwestling** [comment]
Good idea. One could even considering putting an ACL token there.... and then skip place of home token when ACL floats.
